CommonTools/Utils/interface/TMVAEvaluator.h

```diff
@@ -0,0 +1,36 @@
+#ifndef CommonTools_Utils_TMVAEvaluator_h
+#define CommonTools_Utils_TMVAEvaluator_h
+
+#include <memory>
+#include <string>
+#include <vector>
+#include <map>
+
+#include <TROOT.h>
+#include "TMVA/Factory.h"
+#include "TMVA/Tools.h"
+#include "TMVA/Reader.h"
```

Comment:
only this include is needed here in the header, at least visually.
Can the other 3 be removed?